connector/src/main/scala/com/microsoft/kusto/spark/common/KustoOptions.scala

```diff
@@ -25,31 +30,34 @@ trait KustoOptions {
 
   val KUSTO_ACCESS_TOKEN: String = newOption("accessToken")
 
-  // Use only for local runs where one can open the browser
+  // Use only for local runs where one can open the browser - set to true to enable
   val KUSTO_USER_PROMPT: String = newOption("userPrompt")
   // AAD application pfx certificate path
   val KUSTO_AAD_APP_CERTIFICATE_PATH: String = newOption("kutoAadAppCertPath")
   // AAD application certificate password
   val KUSTO_AAD_APP_CERTIFICATE_PASSWORD: String = newOption("kutoAadAppCertPassword")
 
+  // KeyVault options. Relevant only if credentials need to be retrieved from Key Vault
+  val KEY_VAULT_URI = "keyVaultUri"
+  val KEY_VAULT_APP_ID = "keyVaultAppId"
+  val KEY_VAULT_APP_KEY = "keyVaultAppKey"
+
   // Classpath to a class that its constructor accepts one argument of type CaseInsensitiveMap[String] which will contain
   // the options provided to the connector. The class should extend Callable[String] with Serializeable and is expected
   // to return an AAD token upon invoking the call method.
   // The provider will be called for every request to the kusto service
   val KUSTO_TOKEN_PROVIDER_CALLBACK_CLASSPATH: String = newOption("tokenProviderCallbackClasspath")
 
-  // Target/source Kusto cluster for writing/reading the data.
-  val KUSTO_CLUSTER: String = newOption("kustoCluster")
-  // Target/source Kusto database for writing/reading the data. See KustoSink.md/KustoSource.md for
-  // required permissions
-  val KUSTO_DATABASE: String = newOption("kustoDatabase")
+  /** Optional parameters */
+  // Kusto ingestion uri cluster for reading data - provide this if ingestion uri cannot be deducted from adding
+  // "ingest-" prefix to the KUSTO_CLUSTER provided. i.e when using proxies.
+  val KUSTO_INGESTION_URI: String = newOption("kustoIngestionUri")
```

Comment:
This is a URL

Reply:
we call it URI everywhere

Reply:
I'd argue that we inconsistently vacillate between URL and URI:
![image](https://user-images.githubusercontent.com/30245027/182030218-93e183b5-8a0f-4abb-8756-dad290321429.png)

In which case when creating a new variable, might as well do it right IMHO.

Reply:
Maybe its a PM decision
I prefer to be concise with docs and other noteable instances i looked

Reply:
Should I ask the PM, or will you?

If you do it, I suggest including the difference between the 2 in your question. For example:
From https://www.geeksforgeeks.org/difference-between-url-and-uri, "URL is used to describe the identity of an item. URI provides a technique for defining the identity of an item. URL links a web page, a component of a web page or a program on a web page with the help of accessing methods like protocols. URI is used to distinguish one resource from other regardless of the method used. URL is a type of URI. URI is the superset of URL. URL is comprised of protocol, domain, path, hash, and so on. URI is comprised of scheme, authority, path, query and many more. Example URL: https://www.geeksforgeeks.org/ Example URI: isbn:0-294-56559-3"

I think that's important to be consistent, but I think it's more important to be accurate. Here, "URL" is more accurate (and consistent as well because we use both "URL" and "URI" terminology).